HLTrigger/Configuration/python/customizeHLTforPatatrack.py

```diff
@@ -52,12 +66,20 @@ def customise_cpu_pixel(process):
     )
 
     from RecoPixelVertexing.PixelTriplets.caHitNtupletCUDA_cfi import caHitNtupletCUDA as _caHitNtupletCUDA
-    process.hltPixelTracksSoA = _caHitNtupletCUDA.clone(
+    process.hltPixelTracksHitQuadruplets = _caHitNtupletCUDA.clone(
         idealConditions = False,
         pixelRecHitSrc = "hltSiPixelRecHitSoA",
         onGPU = False
     )
 
+    process.hltPixelTracksSoA = cms.EDAlias(
+        hltPixelTracksHitQuadruplets = cms.VPSet(
+            cms.PSet(
+                type = cms.string("32768TrackSoATHeterogeneousSoA")
```

Comment:
I'm wondering `32768` here. Is the C++ type `HeterogeneousSoA<pixelTrack::TrackSoA>`? `edmToFriendlyClassName "HeterogeneousSoA<pixelTrack::TrackSoA>"` gives `pixelTrackTrackSoAHeterogeneousSoA`.

Reply:
```python
process.ascii = cms.OutputModule("AsciiOutputModule",
    verbosity = cms.untracked.uint32(2),
    outputCommands = cms.untracked.vstring(
        'drop *',
        'keep *_hltPixelTracksHitQuadruplets_*_*'
    )
)
```
gives
```
Branch Type = Event
Process Name = HLTGRun
ModuleLabel = hltPixelTracksHitQuadruplets
Branch ID = 1538655069
Class Name = HeterogeneousSoA<TrackSoAT<32768> >
Friendly Class Name = 32768TrackSoATHeterogeneousSoA
Product Instance Name =
```

which agrees with
```bash
$ edmToFriendlyClassName 'HeterogeneousSoA<TrackSoAT<32768>>'
32768TrackSoATHeterogeneousSoA
```

I guess `edmToFriendlyClassName` clearly cannot know about template parameters with default values ?

Reply:
Thanks, I didn't look carefully enough then. The `edmToFriendlyClassName` does only text manipulation, so if the C++ type is wrong, so is the output friendly name :)